Mathlib/Data/Option/Basic.lean

```diff
@@ -319,6 +319,10 @@ theorem orElse_none' (x : Option α) : x.orElse (fun _ ↦ none) = x := by cases
 
 #align option.ne_none_iff_is_some Option.ne_none_iff_isSome
 
+@[simp]
```

Comment:
Flagging this for fellow reviewers to note: this lemma is new (but very similar to exist ones).

Reply:
Let me revert this `@[simp]` attr and move it to a new PR.
```suggestion
```